docs/resources/bridge_domain.md

```diff
@@ -7,116 +10,407 @@ description: |-
+  - Default: `yes`
+  - Valid Values: `no`, `yes`.
+* `pim_ipv6` (ipv6McastAllow) - (string) Enable IPv6 multicast traffic for the Bridge Domain object.
+  - Default: `no`
+  - Valid Values: `no`, `yes`.
+* `limit_ip_learn_to_subnets` (limitIpLearnToSubnets) - (string) Limit IP address learning to subnets for the Bridge Domain object. Every Bridge Domain object can have multiple subnets associated with it.
+  - Default: `yes`
+  - Valid Values: `no`, `yes`.
+* `link_local_ipv6_address` (llAddr) - (string) The override of the system generated IPv6 link-local address.
+* `custom_mac_address` (mac) - (string) The MAC address of the Bridge Domain object.
+  - Default: `00:22:BD:F8:19:FF`
+* `drop_arp_with_multicast_smac` (mcastARPDrop) - (string) Drop rogue multicast ARP packets for the Bridge Domain object.
+  - Default: `yes`
+  - Valid Values: `no`, `yes`.
+* `pim` (mcastAllow) - (string) Enable IPv4 multicast traffic for the Bridge Domain object.
+  - Default: `no`
+  - Valid Values: `no`, `yes`.
+* `multi_destination_flooding` (multiDstPktAct) - (string) The multiple destination forwarding method for L2 Multicast, Broadcast, and Link Layer traffic types.
+  - Default: `bd-flood`
+  - Valid Values: `bd-flood`, `drop`, `encap-flood`.
+* `name_alias` (nameAlias) - (string) The name alias of the Bridge Domain object.
+* `owner_key` (ownerKey) - (string) The key for enabling clients to own their data for entity correlation.
+* `owner_tag` (ownerTag) - (string) A tag for enabling clients to add their own data. For example, to indicate who created this object.
+* `bridge_domain_type` (type) - (string) domain type.
+  - Default: `regular`
+  - Valid Values: `fc`, `regular`.
+* `unicast_routing` (unicastRoute) - (string) Enables L3 routing and endpoint IP learning for the Bridge Domain object.
+  - Default: `yes`
+  - Valid Values: `no`, `yes`.
+* `l2_unknown_unicast_flooding` (unkMacUcastAct) - (string) The forwarding method for unknown layer 2 destinations.
+  - Default: `proxy`
+  - Valid Values: `flood`, `proxy`.
+* `l3_unknown_multicast_flooding` (unkMcastAct) - (string) The forwarding method for unknown layer 3 multicast destinations.
+  - Default: `flood`
+  - Valid Values: `flood`, `opt-flood`.
+* `ipv6_l3_unknown_multicast_flooding` (v6unkMcastAct) - (string) The forwarding method for unknown IPv6 multicast destinations.
+  - Default: `flood`
+  - Valid Values: `flood`, `opt-flood`.
+* `virtual_mac_address` (vmac) - (string) The virtual MAC address of the Bridge Domain object. This is used when the the BD/SVI is extended to multiple sites using a L2 Outside.
+  - Default: `not-applicable`
 
-* `relation_fv_rs_bd_to_profile` - (Optional) Relation to L3Outs Route Map For Import and Export Route Control (Point to class rtctrlProfile). Cardinality - N_TO_ONE. Type - String.
+* `legacy_mode` - (map) A map of Legacy Mode (ACI object [fvAccP](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvAccP/overview)).
```

Comment:
The structure we have now is that all children are either maps or set of maps when multiple are configurable. If we want to change this the generator should be changed. If you think we should go this route we should have a discussion with the team about this.

Reply:
Yeah I thought it would depend on how much extra complexity we are adding to the generator to catch these single attribute cases. Happy to leave it as is to reduce possibility of breaking changes.